### DISCUSSION TITLE
Was the lecture (5.11.21) not recorded?

### DISCUSSION THREAD

**Discussion by MartinBilko** [irgroup-classrooms/dis17-2021#14] [Q&A] [answered]
Just a reminder that the video still needs to be linked if it exists.

**Comment by phschaer** [comment, marked as answer]
Dang! Sorry, I forgot to upload this one. Will be ready in a few minutes.

**Reply by phschaer** [reply]
And here it is: https://th-koeln.sciebo.de/s/D8DZUZHCmVUMb7E
